components/tidb_query/src/expr_util/conv.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use crate::codec::data_type::*;
+
+#[derive(Copy, Clone)]
+struct IntWithSign(u64, bool);
+
+impl IntWithSign {
+    fn from_int(num: Int) -> IntWithSign {
+        IntWithSign(num.wrapping_abs() as u64, num < 0)
+    }
+
+    fn from_signed_uint(num: u64, is_neg: bool) -> IntWithSign {
+        IntWithSign(num, is_neg)
+    }
+
+    // Shrink NUM to fit the boundary of i64.
```

Comment:
maybe NUM --> `num` is better?

Reply:
Ok, I didn't find some rules to guide this in CODE_COMMENT_STYLE.md. So I follow the convention in some Lisp project -- using UPPER CASE to identify the argument.